.github/workflows/plugin_test.yaml

```diff
@@ -28,13 +28,13 @@ jobs:
             branch: main
             tests_to_run: tests/.
           - plugin: pynxtools-mpes
-            branch: main
+            branch: update-to-new-nxmpes
```

Comment:
Just a reminder to update this before merging

Reply:
done